clutch/src/main.rs

```diff
@@ -6,9 +6,18 @@ use lurk::eval::lang::{Coproc, Lang};
 use lurk::field::LanguageField;
 use lurk::repl::repl_cli;
 use pasta_curves::pallas;
+use tracing_subscriber::{fmt, prelude::*, EnvFilter, Registry};
+use tracing_texray::TeXRayLayer;
 
 fn main() -> Result<()> {
-    pretty_env_logger::init();
+    let subscriber = Registry::default()
+        // TODO: correctly filter log level with `clap_verbosity_flag`
+        .with(fmt::layer().pretty())
+        .with(EnvFilter::from_default_env())
+        // note: we don't `tracing_texray::examine` anything below, so no spans are printed
+        // but we add the layer to allow the option in the future, maybe with a feature?
+        .with(TeXRayLayer::new());
```

Comment:
I don't expect we would be driving our micro-benchmarking from clutch, so perhaps the TexRay layer is superfluous in the clutch crate entirely?